### PR TITLE
chore: fix PHP CS Fixer configuration

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see comments.

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -56,9 +49,5 @@
         'ternary_to_null_coalescing'               => true,
         'use_arrow_functions'                      => true,
         'void_return'                              => true,
-    ])->setFinder(
-        PhpCsFixer\Finder::create()
-            ->exclude('vendor')
-            ->in(__DIR__)
-    )
+    ])->setFinder($finder)
```
Comment (szepeviktor) [inline .php-cs-fixer.dist.php:52]: Removing double finder.

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -3,17 +3,10 @@
...
-            'node_modules',
-            'public',
-            'templates',
-            'var',
-        ]
-    )
+    ->in([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+    ])
```
Comment (szepeviktor) [inline .php-cs-fixer.dist.php:9]: Explicit is better than implicit.

File: .php-cs-fixer.dist.php (hunk 1 above)
Comment (hypnotox) [reply .php-cs-fixer.dist.php:52]: That i missed that is a bit embarrassing lol, nice catch

Comment (szepeviktor) [reply .php-cs-fixer.dist.php:52]: Do not be embarrassed. I've been training my eyes to spot errors in the last 10 years (I'm not writing code)

**Pull request merged**